docs/content/connectors/oracle-cdc.md

```diff
@@ -96,6 +96,7 @@ You have to enable log archiving for Oracle database and define an Oracle user w
      GRANT EXECUTE_CATALOG_ROLE TO flinkuser;
      GRANT SELECT ANY TRANSACTION TO flinkuser;
      GRANT LOGMINING TO flinkuser;
+     GRANT ANALYZE ANY TO flinkuser;
```

Comment:
This seems to require greater permission, Why do we need to add this？

Reply:
@lvyanquan `analyze table` statement need this permission.https://github.com/ververica/flink-cdc-connectors/blob/9ce36569fb98fbfa4b78178277ecc8230e9ea049/flink-cdc-connect/flink-cdc-source-connectors/flink-connector-oracle-cdc/src/main/java/com/ververica/cdc/connectors/oracle/source/utils/OracleUtils.java#L71